### ISSUE TITLE
Cannot open OGR Source: WFS:http://geodata.nationaalgeoregister.nl/rdinfo/wfs?

### ISSUE THREAD

**Issue by frafra** [geopython/pygeoapi#1013]
I am testing the pygeoapi Docker image, and I got this messages:

```
START /entrypoint.sh
Trying to generate openapi.yml
Error Number: 1, Type: Failure, Msg: HTTP error code : 404
Error Number: 1, Type: Failure, Msg: Error returned by server : HTTP error code : 404 (0)
Cannot open OGR Source: WFS:http://geodata.nationaalgeoregister.nl/rdinfo/wfs?
Cannot open OGR Source: WFS:http://geodata.nationaalgeoregister.nl/rdinfo/wfs?
Error Number: 1, Type: Failure, Msg: HTTP error code : 400
Error Number: 1, Type: Failure, Msg: Error returned by server : HTTP error code : 400 (0)
Error Number: 1, Type: Failure, Msg: HTTP error code : 400
openapi.yml generated continue to pygeoapi
```

It looks there is a problem with the URL.
https://github.com/geopython/pygeoapi/blob/42a419299b714be66f2f862f87a7503efe24d235/pygeoapi/provider/ogr.py#L88

**Comment by justb4** [comment]
Yes, hold on, this URL was changed last week. Will get back.Just @ MobileOn Oct 13, 2022 18:20, Francesco Frassinelli ***@***.***> wrote:
I am testing the pygeoapi Docker image, and I got this messages:
START /entrypoint.sh
Trying to generate openapi.yml
Error Number: 1, Type: Failure, Msg: HTTP error code : 404
Error Number: 1, Type: Failure, Msg: Error returned by server : HTTP error code : 404 (0)
Cannot open OGR Source: WFS:http://geodata.nationaalgeoregister.nl/rdinfo/wfs?
Cannot open OGR Source: WFS:http://geodata.nationaalgeoregister.nl/rdinfo/wfs?
Error Number: 1, Type: Failure, Msg: HTTP error code : 400
Error Number: 1, Type: Failure, Msg: Error returned by server : HTTP error code : 400 (0)
Error Number: 1, Type: Failure, Msg: HTTP error code : 400
openapi.yml generated continue to pygeoapi

It looks there is a problem with the URL.
https://github.com/geopython/pygeoapi/blob/42a419299b714be66f2f862f87a7503efe24d235/pygeoapi/provider/ogr.py#L88
